### ISSUE TITLE
Improvement for 📢Announcements

### ISSUE THREAD

**Issue by TsukiZero** [FrankerFaceZ/FrankerFaceZ#1214]
**Is your feature request related to a problem? Please describe.**
Unrelated to a problem.

**Describe the solution you'd like**
The idea is to add different ways to handle the in-chat announcements (which uses the div class "announcement-line") to make them take less space in chat and be given importance by another mean.
Options could include:
_Pin most recent on top of the chat_ - (Checkbox) Shows the most recent announcement on the top of the chat (similar to Hype Train, Poll, Predictions, and other notifications)
_"Show/Hide Announcements" button_ - (Checkbox) Adds a button to show or hide announcements right next to ⚙️ Chat Settings.
_Separate Announcements from chat_ - (List) Provides three options on how to split Announcements from chat: "None", "Top Bar (Exclusive)" (adds a new Notifications button for currently visible chat announcements), "Top Bar (Notifications)" (creates a new tab under Top Bar's "Notification" button to show currently visible chat announcements).

**Describe alternatives you've considered**
None, maybe Stylebot to hide it, but I don't want to exclusively hide it.

**Comment by furai** [comment]
Yeah, I'd love to make it so announcements look like a regular message. I don't need yet another attention grabbing thing in my chat.
